http.go

```diff
@@ -343,7 +350,10 @@ func wsHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func uiHandler(w http.ResponseWriter, _ *http.Request) {
-	data, err := staticContent.ReadFile(path.Join("res", "tmpl", "index.html"))
+	f, _ := staticContent.Open(path.Join("res", "tmpl", "index.html"))
+	fInfo, _ := f.Stat()
+	var data = make([]byte, fInfo.Size())
+	_, err := f.Read(data)
```

Comment:
Is it possible to simplify this by using [`fs.ReadFile()`](https://pkg.go.dev/io/fs#ReadFile)? It should behave much like the old ReadFile method, only you will need to pass in staticContent as the first parameter 🙂

Reply:
Ah, that's much nicer. I tried doing staticContent.ReadFile() but it wasn't on the interface, thanks for pointing this out.